[Doc] [Do-not-Merge] Add a parameter to forcefully close fenced topics (#14289)
apache/pulsar: site2/docs/reference-configuration.md

|isAllowAutoUpdateSchemaEnabled|Allow schema to be auto updated at broker level.|true|
|schemaCompatibilityStrategy| The schema compatibility strategy at broker level, see [here](schema-evolution-compatibility.md#schema-compatibility-check-strategy) for available values.|FULL|
|systemTopicSchemaCompatibilityStrategy| The schema compatibility strategy is used for system topics, see [here](schema-evolution-compatibility.md#schema-compatibility-check-strategy) for available values.|ALWAYS_COMPATIBLE|

> | topicFencingTimeoutSeconds | If a topic remains fenced for a certain time period (in seconds), it will be closed forcefully. If set to 0 or a negative number, the fenced topic will not be closed. | 0 |

Reviewer: ```suggestion
| topicFencingTimeoutSeconds | If a topic remains fenced for a certain time period (in seconds), it is closed forcefully. If set to 0 or a negative number, the fenced topic is not closed. | 0 |
```

Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp
